test/e2e/manifest_test.go

```diff
@@ -49,6 +49,16 @@ var _ = Describe("Podman manifest", func() {
 		Expect(session.ExitCode()).To(Equal(0))
 	})
 
+	It("podman manifest inspect", func() {
+		session := podmanTest.Podman([]string{"manifest", "inspect", BB})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+
+		session = podmanTest.PodmanNoCache([]string{"manifest", "inspect", "docker.io/library/busybox"})
```

Comment:
```suggestion
		session = podmanTest.PodmanNoCache([]string{"manifest", "inspect", "quay.io/library/busybox"})
```
Is this possible?

Reply:
It's possible to inspect the manifest from quay.io, but I guess there's no library/busybox repository, do you have other suggestions?

Reply:
@QiWang19 I did a grep of the tests directory and found other tests using `quay.io/libpod/alpine_labels:latest` does that meet your needs?

Reply:
`quay.io/libpod/alpine_labels:latest` is not a manifest list type